src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/sidepanel.properties

```diff
@@ -0,0 +1 @@
+delete.cloud=Delete the cloud?
```

Comment:
Not sure why you changed the original text that was referring to the cloud name?

Reply:
Looks like a core bug with the quote
```suggestion
delete.cloud=Delete the cloud ''{0}'' ?
```
should work as expected.

Reply:
Actually in core there is no problem because the properties file is UTF-8 encoded.